src/lexer.coffee

```diff
@@ -186,7 +186,10 @@ exports.Lexer = class Lexer
       else if tag is 'UNLESS'
         tag = 'IF'
       else if tag is 'IMPORT'
-        @seenImport = yes
+        if match.input.match /^import\s*\.\s*[_a-zA-Z$][_a-zA-Z$0-9]*/
```

Comment:
Rather than "lexing ahead" via regex (which is best avoided if possible), I'd try and mimic how eg `NEW_TARGET` works and "retroactively adjust" to update a previously seen `IMPORT` token to instead be an `IMPORT_META` token

So ya if you grep `NEW_TARGET` in `src/lexer.coffee` you can see the pattern for that

Reply:
Done!